SparseGrids/tsgGridFourier.hpp

```diff
@@ -183,6 +189,11 @@ class GridFourier : public BaseCanonicalGrid {
     MultiIndexSet tensors;
     MultiIndexSet active_tensors;
     std::vector<int> active_w;
+
+    MultiIndexSet updated_tensors;
```

Comment:
Leave the variables out of this
* having unused variables will generate an unnecessary warning
* the variables should be added to the file I/O, (make another PR with just variables and file I/O)

Reply:
Done